tools/CMakeLists.txt

```diff
@@ -5,5 +5,11 @@ find_package(Threads)
 add_executable(name name.cpp)
 target_link_libraries(name abieos ${CMAKE_THREAD_LIBS_INIT})
 
+add_executable(generate_hex_from_json util_generate_hex_from_json.cpp ../src/abieos.cpp)
```

Comment:
Seems you should link with `abieos` instead of compiling in abieos.cpp directly.

Reply:
good point, thanks